pkg/configauditreport/controller/controller.go

```diff
@@ -199,7 +201,7 @@ func (r *ResourceController) reconcileResource(resourceKind kube.Kind) reconcile
 
 		// Skip processing if there are no policies applicable to the resource
 
-		applicable, reason, err := policies.Applicable(resource)
+		applicable, reason, err := policies.Applicable(resource, r.RbacAssessmentScannerEnabled)
```

Comment:
Maybe pass `r.RbacAssessmentScannerEnabled` to `NewPolicies`? Because once the scanner is enabled, no need to recheck every time?

Reply:
not sure how it help we still need to do this check for every resource reconcile and in other places (reconcileConfig) where we call new policies `RbacAssessmentScannerEnabled` is not used